ckanext/envvars/tests.py

```diff
@@ -1,12 +1,16 @@
 import os
-from nose.tools import assert_equal
-from nose.plugins.skip import SkipTest
+import pytest
 
-from pylons import config
-import ckan
 import ckan.plugins.toolkit as toolkit
 import ckan.plugins as p
 
+try:
+    p.toolkit.requires_ckan_version("2.8")
+except p.toolkit.CkanVersionException:
+    from pylons import config
+else:
+    from ckan.common import config
```

Comment:
For plugins that need to work across different CKAN versions is best to use the [ckantoolkit](https://github.com/ckan/ckantoolkit) package, which wraps toolkit objects to make them available in different CKAN versions. Add it as a requirement to `setup.py` ( using `install_requires=["ckantoolkit"]`, you can leave it unpinned)

```suggestion
from ckantoolkit import config
```

You can replace all other usage of the toolkit with

```python

import ckantoolkit as toolkit
```